script.js

```diff
@@ -0,0 +1,125 @@
+let quizData;
+let quizNum = 1;
+let answerNum = 0;
+const quizAllNum = 10;
+
+const startQuiz = () => {
+
+	fetch('https://opentdb.com/api.php?amount=10')
+	.then(function(response) {
+		return response.json();
+	})
+	.then(function(myJson) {

```

Comment:
ここもアロー関数を利用した方が、他の部分と統一されて見やすいと思います。
```suggestion
	.then(function(myJson) {
	.then((myJson) => {
```

Reply:
承知しました。修正いたします。